src/core/api/types/stockOperation/mocks/StockOperationDTO.mock.ts

```diff
@@ -0,0 +1,104 @@
+import { StockOperationDTO } from "../StockOperationDTO";
+import { RecordPermission } from "../../RecordPermission";
+export const mockStockOperationDTO: StockOperationDTO = {
+  receivedItems: [],
```

Comment:
receivedItems is not part of a stock operation creation.It actually fails an operation from being created

Reply:
I also encourage everyone to take time and test their changes to avoid the loop we are getting into where things work and the next minute they are not. Thanks cc. @jabahum @hadijahkyampeire @donaldkibet

Reply:
noted